CONTRIBUTIONS.md

```diff
@@ -0,0 +1,87 @@
+# 🧑‍💼Contributing to Deakin Detonator Toolkit
+
+Thank you for your interest in contributing to the Deakin Detonator Toolkit aka PT-GUI project! This project is student run and we welcome contributions from the students of Deakin to enhance this penetration testing toolkit. Please follow the guidelines below to ensure a smooth contribution process.
+
+## Table of Contents
+
+-   [Getting Started](#getting-started)
+-   [How to Contribute](#how-to-contribute)
+-   [Coding Standards](#coding-standards)
+-   [Submitting Contributions](#submitting-contributions)
+-   [Reporting Issues](#reporting-issues)
+
+## 👶Getting Started
+
+To get started with contributing, follow these steps; If you're not familiar with using Git team recommends you to first understand how the **[Git](https://deakin365.sharepoint.com/:w:/s/HardhatEnterprises2/Ee1Nlb_OZDVIlElD5RBO5o0BzG0OTRTFHIhWpTNwaxthmQ?e=Bg0rTb)** works with **[GitHub](https://deakin365.sharepoint.com/:b:/s/HardhatEnterprises2/ETU_JVpc67BCqgp6W7vm9ewBzbWCnlneQc-mzhiOIwGd8g?e=2pW0ld)**.
+
+1. **Clone the Repository**:
+
+    ```bash
+    git clone https://github.com/Hardhat-Enterprises/Deakin-Detonator-Toolkit.git
+    cd ~/path/to/Deakin-Detonator-Toolkit
+
+    ```
+
+2. **Set Up the Development Environment**:
+   Follow the instructions in the **[README](https://github.com/Hardhat-Enterprises/Deakin-Detonator-Toolkit/blob/main/README.md#-setup)** to set up your environment and run the application locally.
+
+## ✍️How to Contribute
+
+We welcome various types of contributions, including:
+
+-   **Bug Fixes**: Identify and fix bugs in the codebase.
+-   **New Features**: Propose and implement new features or enhancements.
+-   **Documentation Improvements**: Help improve project documentation and architectural decision record.
+-   **Testing**: Perform testing on the appilcation to ensure code reliability and stability.
+
+## ⚠️Coding Standards
+
+To maintain code quality, please adhere to the following coding standards:
+
+-   Use consistent indentation (**2 spaces** for JavaScript/TypeScript).
+-   Follow naming conventions (**camelCase** for variables and functions).
+-   Write clear and concise comments where necessary.
+-   Ensure that your code passes linting checks.
+    -   Perform the following commands while committing the code:
+    -   ```bash
+           yarn run style
+           npx prettier -w file-name
```

Comment:
Add a note that `npx prettier -w file-name` will overwrite its changes to `file-name`, so it's recommended to always take a backup of `file-name` before running this command. May also be worth noting that `npx prettier` only needs to be run if `yarn run style` has an issue.

Reply:
Hi Cam, I have updated this section with additional note.